wdl-lint/tests/lints/unknown_rule/source.wdl

```diff
@@ -1,6 +1,13 @@
+#@ except: SnackCase
+
 version 1.1
 
-#@ except: ThisIsNotARealRule, MissingMetas
```

Comment:
please put this line back in

Reply:
See the above suggestion @mrswastik-robot